src/components/MapDisplay/MapDisplay.js

```diff
@@ -3,35 +3,103 @@ import React, { Component } from 'react'
 import './MapDisplay.css'
 
 const meterConversion = 1609.34;
+const CONSTANTS = {
+  tileURL: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  attribution: 'Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors',
+  minZoom: 4,
+  maxZoom: 20,
+  defaultZoom: 10,
+  meterConversion: 1609.34
+};
+const styles = {
+  red: {
+    color: 'red',
+    fillColor: '#f03',
+    fillOpacity: 0.5
+  }
+};
 
 class MapDisplay extends Component {
-    
+
     componentDidMount () {
-        const {
+        let {
```

Comment:
these should stay as const no need for let.

Reply:
Oops I dunno why I changed that. I'll change it back